### DISCUSSION TITLE
How to use List -> FocusLost()

### DISCUSSION THREAD

**Discussion by junmaqiang** [fyne-io/fyne#3580] [Q&A] [answered]
## I don't want the rows of the List to be selected, I'm using widget.NewListWithData()
![WX20230118-144526](https://user-images.githubusercontent.com/1819457/213102882-53624827-b18c-43a3-9de1-7ca15c915542.png)

## I don't know how to use this method
![WX20230118-144707](https://user-images.githubusercontent.com/1819457/213103195-e79e1077-e0f7-4953-995b-932c72648afc.png)

## I looked at the test code, but I can't use it yet,  fyne V2.3.0 is no test case for widget.NewListWithData()
![WX20230118-145023](https://user-images.githubusercontent.com/1819457/213103766-414c1d70-c56f-422e-baf9-e5542b391b35.png)

**Comment by Jacalz** [comment, marked as answer]
You should not call the `FocusLost()` method as that is what the focus manager calls to tell the widget that it has lost focus.

What you want to do is to make the list's `.OnSelected(id ListItemID)` function (https://pkg.go.dev/fyne.io/fyne/v2/widget#List) call `.Unselect(id)` (https://pkg.go.dev/fyne.io/fyne/v2/widget#List.Unselect) on the list.
